go/runtime/client/api/api.go

```diff
@@ -122,6 +125,21 @@ type GetTxsRequest struct {
 	IORoot    hash.Hash        `json:"io_root"`
 }
 
+// GetEventsRequest is a GetEvents request.
+type GetEventsRequest struct {
+	RuntimeID common.Namespace `json:"runtime_id"`
+	Round     uint64           `json:"round"`
+}
+
+// Event is an event emitted by a runtime.
+//
+// Key and value semantics are runtime-dependent.
```

Comment:
Should there be a note that these are basically the "runtime transaction tags"?

Reply:
Added.